lib/bulk_data_test_kit/v2.0.0/client/docs/suite_description.md

```diff
@@ -0,0 +1,25 @@
+The Bulk Data Access v2.0.0 Client test suite validates the conformance of a client
+to the [FHIR Bulk Data Access IG STU2](http://hl7.org/fhir/uv/bulkdata/STU2).
+
+## Scope
+
+These tests are a **DRAFT** intended to allow implementers to perform
+preliminary checks of their systems against the requirements stated for Bulk Data client actors
+and [provide feedback](https://github.com/inferno-framework/bulk-data-test-kit/issues)
+on the tests. Future versions of these tests may verify other
+requirements and may change the test verification logic.
+
+### Sample Execution - Postman
+
+To try out these tests without a Bulk Data client implementation, you may
+run them using [this "bulk data client system export" Postman collection](https://github.com/inferno-framework/bulk-data-test-kit/blob/main/lib/bulk_data_test_kit/v2.0.0/client/postman/system_export.postman_collection.json) and [this "bulk data client delete" Postman collection](https://github.com/inferno-framework/bulk-data-test-kit/blob/main/lib/bulk_data_test_kit/v2.0.0/client/postman/delete.postman_collection.json).
+
+To run client tests against one of the Postman collections:
+1. Start an Inferno session of the Bulk Data Client test suite.
+2. Navigate to either the export tests group or the delete tests group (depending on which Postman collection you want to use).
+3. Click the "Run Tests" button in the upper right and select the "System Level Export" option for export type.
```

Comment:
It looks like the first Bulk Data Client Export Tests group fails when selecting the "System Level Export" option for the export type since the request is defined as a Group export in the Postman collection. Either update the postman collection to be a system export, or update the documentation to say the user should choose the Group export option so that all the tests pass for both test groups